docs/product-assembly_running_employee_rostering_in_wb/src/main/asciidoc/wb-employee-rostering-register-solver-proc.adoc

```diff
@@ -0,0 +1,32 @@
+[id='wb-employee-rostering-register-solver-proc']
+= Register the Solver using the REST API 
+
+Once the Solver has been created, you can register it using the REST API. By registering the Solver, you can then submit planning problems in the request body to find the optimal solution.
+
+.Prerequisite
+
+* The employee rostering project has been set up and deployed according to the previous chapters in this guide.
+* The Solver has been configured as instructed in <<wb-employee-rostering-solver-configuration-proc>> and <<wb-employee-rostering-solver-termination-configuration-proc>>.
+* You have successfully built and deployed the employee rostering project.
+
+.Procedure 
+. Create a HTTP request using the following header:
++
+[source]
+----
+authorization: Basic ZG1BZG1phjphZG1pbg==
```

Comment:
Are you sure this will work?
```
$ echo -n ZG1BZG1phjphZG1pbg== | base64 -d
dmAdmi�:admin
```
on the other hand:
```
$ echo -n admin:admin | base64
YWRtaW46YWRtaW4=
```

In any case, I think we should also mention that in this specific example the value represents the `admin:admin` credentials and the prerequisite for this to work is that the user has the `kie-server` role already covered here (https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.0/html/installing_and_configuring_red_hat_business_optimizer/optimizer-rest-api-configuration-ref).